yacht/utils.py

```diff
@@ -32,6 +33,10 @@ def load_signature_with_ksize(filename: str, ksize: int) -> sourmash.SourmashSig
     sketches = list(sourmash.load_file_as_signatures(filename, ksize=ksize))
     if len(sketches) != 1:
         raise ValueError(f"Expected exactly one signature with ksize {ksize} in {filename}, found {len(sketches)}")
+    if len(sketches[0].minhash.hashes) == 0:
+        raise ValueError(f"Empty hash in signature. Please try sketch with '--scaled=1' or use alternative to sourmash.")
```

Comment:
One quibble: let's change the error message to something like: "Empty sketch in signature. This may be due to too high of a scale factor, please reduce it (eg. `--scaled=1`) and try again."